carmajava/rsumetering/src/main/java/gov/dot/fhwa/saxton/carma/rsumetering/CommandingState.java

```diff
@@ -32,6 +32,15 @@
  * Set the target vehicle speed to the platoon speed and activate the lane change indicator when needed
  */
 public class CommandingState extends RSUMeteringStateBase {
+  protected static final double MIN_METER_DIST = 0.0; //meters
+  protected static final double MAX_METER_DIST = 1000.0; //meters
+  protected static final double MIN_MERGE_DIST = 5.0; //meters
```

Comment:
This value needs to be able to go negative by at least the length of the merge area. The merge distance is the distance to the start of the merge region if I recall correctly.

Reply:
done.